packages/storybook-css/src/Table.stories.tsx

```diff
@@ -1910,7 +1910,9 @@ export const FigureCaption: Story = {
   render: (({ descriptionId, children }: PropsWithChildren<{ descriptionId: string }>) => (
     <figure>
       <Table aria-describedby={descriptionId}>{children}</Table>
-      <figcaption id={descriptionId}>Bron: Gemeente Utrecht</figcaption>
+      <figcaption className="utrecht-table__figcaption" id={descriptionId}>
+        Bron: Gemeente Utrecht
+      </figcaption>
```

Comment:
I would want to make the `figcaption` optional and make the text "Bron: Gemeente Utrecht" configurable from the properties.